arches/app/templates/views/components/search/advanced-search.htm

```diff
@@ -122,23 +122,42 @@
                         'aria-controls': uid,
                         'aria-label': $root.translations.resourceTypeSearchFacets(name),
                     }
-                ">
+                    ">
                     <div class="search-facet-item-heading">
                         <strong data-bind="text: name"></strong>
                         <span><i class="fa report-expander print-hide" data-bind="css: {'fa-angle-down': !collapsed(), 'fa-angle-right': collapsed()}"></i></span>
                     </div>
                 </div>
                 <div data-bind="visible: !collapsed()" role="list">
                     <!-- ko foreach: cards -->
-                    <div role="listitem">
-                        <a href="#" class="search-facet-item" role="button" data-bind="onEnterkeyClick, onSpaceClick, 
-                            click: addFacet, 
-                            attr: {
-                                id: uid,
-                                'aria-label': $root.translations.addCardToSearchQuery(name, $parent.name)
-                            }">
-                            <p class="search-facet-item-heading" data-bind="text: name"></p>
-                        </a>
+                    <div role="listitem" style="display: flex; align-items:center;">
+                        <div class="search-facet-item" style="display: flex;">
+                            <div>
+                                <div style="margin-right:5px;" role="button" data-bind="onEnterkeyClick, onSpaceClick, click: self.getNodegroupPath, class: $data.nodegroup_id == self.selectedNodegroupId() && self.expandPath() ? 'fa fa-times-circle' : 'fa fa-info-circle'"></div>
```

Comment:
Strict equality `===`

Also, since this icon has semantic meaning, can you add an aria-label?